extension/src/cli/dvc/constants.ts

```diff
@@ -1,7 +1,7 @@
 import { join } from 'path'
 
 export const MIN_CLI_VERSION = '2.30.0'
-export const LATEST_TESTED_CLI_VERSION = '2.30.0'
+export const LATEST_TESTED_CLI_VERSION = '2.31.0'
```

Comment:
Along with updating the `LATEST_TESTED_CLI_VERSION`, we need to update our demo project `requirements.txt`. I'll go ahead and open a pr!